### PR TITLE
[AEROGEAR-2421] move from configmap to secret

### PR DESCRIPTION
**Pull request by witmicko**
JIRA:
https://issues.jboss.org/browse/AEROGEAR-2421

Verification steps:
- [x] Successfully provision a new instance of the CRC from this branch and create a new binding
- [x] Verify that no new configmap was created during the bind. 
- [x] Verify that a secret was created and labelled with `serviceInstanceID`, `mobileClientId` and `mobile-client`, and that these labels contain the correct values

@maleck13 @philbrookes @dimitraz @vchepeli

### CODE REVIEW

**Review by maleck13 — CHANGES_REQUESTED** [review]
Am i missing something? Where is the config field with the config data as a string?

**Comment by witmicko** [comment]
@maleck13 it was already there so it is not showing up in the diff 
https://github.com/witmicko/custom-runtime-connector-apb/blob/effc64b660e8c2efa4e99005a7e6662d249cd52b/roles/bind-crc-apb/tasks/main.yml#L45

**Comment by maleck13** [comment]
@witmicko ahh ok. Do you want to add some verification steps @vchepeli might something you can help with here

File: roles/bind-crc-apb/tasks/main.yml [hunk 1]
```diff
@@ -37,7 +37,8 @@
       serviceName: '{{ crc_svc_name }}'
       serviceInstanceID: '{{ _apb_service_instance_id }}'
       mobileClientId: '{{ CLIENT_ID }}'
-    data:
+      mobile-client: "enabled"
```
Comment (maleck13) [inline roles/bind-crc-apb/tasks/main.yml:40]: This should be under labels

**Review by maleck13 — COMMENTED** [review]
minor change plus some verification steps

File: roles/bind-crc-apb/tasks/main.yml (hunk 1 above)
Comment (witmicko) [reply roles/bind-crc-apb/tasks/main.yml:40]: @maleck13 it is, short diff strikes again :) - click on those expand icons to see more of the file
will add verification

**Review by maleck13 — APPROVED** [review]

**Comment by maleck13** [comment]
@witmicko checking this out now

**Comment by maleck13** [comment]
Verified this locally. Have we tagged the previous version and then we can merge
@david-martin  @aliok  FYI we will tag the previous version which should create a tagged image in dockerhub and then latest will be the new client binding flow

**Comment by david-martin** [comment]
@maleck13 A tag of the previous version would be good to have repeatable/reliable steps for demo recording

**Comment by maleck13** [comment]
Yes i have tagged master

**Pull request merged**